doc/nrf/drivers/nrf700x.rst

```diff
@@ -71,6 +71,14 @@ The nRF7002 DK uses QSPI whereas the nRF7002 EK uses SPI.
 
 To connect the nRF7002 EK to the SoC, the ``nrf7002ek`` shield is required.
 
+Configurable options
+********************
+
+The nRF700X Wi-Fi driver has the following configurable options:
+
+.. options-from-kconfig::
```

Comment:
This won't work, as it picks up Kconfig's from `drivers` folder. You need to align the doc directory structure to actual sources for this to work.

Reply:
Done.